index.html

```diff
@@ -0,0 +1,214 @@
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./assets/stylesheets/app.css" />
+    <link rel="stylesheet" href="./assets/stylesheets/index.css" />
+  </head>
+  <body>
+    <header>
+      <div id="nav-banner">
+        <a
+          href="https://www.facebook.com/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a
+          href="https://www.twitter.com"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#">
+          <span>English</span>
+        </a>
+        <a href="#">
+          <span>My Page</span>
+        </a>
+        <a href="#">
+          <span>Logout</span>
+        </a>
+      </div>
+      <nav id="navbar">
+        <i id="menu-btn" class="fa-solid fa-bars"></i>
+        <a href="./" id="logo">
+          <img src="./assets/images/logo.png" alt="logo" />
+        </a>
+        <ul>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section>
+      <div id="mobile-menu" class="d-none flex-column">
+        <header>
+          <div id="close-menu-btn">
+            <i class="fa-solid fa-x"></i>
+          </div>
+        </header>
+        <ul id="mobile-links">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </div>
+    </section>
+
+    <section id="hero">
+      <header id="hero-header">
+        <h3>"Hello World!"</h3>
+        <h1>
+          Aruba Ruby <br>
+          Global summit <br>
+          2023
+        </h1>
+      </header>
+      <article>
+        <p id="sub-theme">
+          Aruba Ruby Global Summit is an annual gathering for all who are
+          passionate about Ruby and Ruby on Rails. The Aruba based conference has served
+          the Ruby community since 2008.
+        </p>
+        <div id="date-venue">
+          <h3 id="date">2023.01.04(THU) ~ 05(FRI)</h3>
+          <h5 id="venue">@Cas di Cultura, Oranjestad, Aruba</h5>
+        </div>
+      </article>
+    </section>
+
+    <section id="program">
+      <header>
+        <h1>Main Program</h1>
```

Comment:
- [ ] It is html best practice to have only one main title in a webpage which you already have one up. Kindly use any of the remaining heading `hx (2, 3, 4, 5, 6).`